minecode/model_utils.py

```diff
@@ -44,7 +44,7 @@
 )
 
 
-def add_package_to_scan_queue(package, pipelines=DEFAULT_PIPELINES, reindex_uri=False, priority=100):
+def add_package_to_scan_queue(package, pipelines=DEFAULT_PIPELINES, priority=0, reindex_uri=False):
     """
     Add a Package `package` to the scan queue to run the list of provided `pipelines`
```

Comment:
Can we add a line about `priority` in the docstring, this is not trivial.

Reply:
docstring updated